styles/components/_modalWindow.mediaQueries.scss

```diff
@@ -0,0 +1,261 @@
+//TO REMOVE!!!!
+html {
+  overflow: hidden;
+}
+
+@media only screen and (max-width: $breakpoint-xxl-max) {
+  .modal {
+    background-color: yellowgreen;
+    &__container {
+      height: calc(100% - 124px);
+      max-width: unset;
+      width: calc(100% - 124px);
+      padding: 32px;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+    }
+    &__heading {
+      font-size: 32px;
+    }
+    &__text {
+      font-size: 24px;
```

Comment:
use rwd set classes. Explanation:
https://discord.com/channels/750077803881431154/781222565921685504/800525746429755402

Reply:
When rwd merge will be concluded

Reply:
The two classes text__md and text__lg are not enough for modal, according to figma there is a very wide variety of fonts on modal. I wil change them into $font-size-primary / some divisor (or rem?). The button is different here than in the rest of the app, so also had to overwrite the main button styling.